### PR TITLE
refactor: changed logger to mcjsonlogger

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: confd/production.tmpl [hunk 1]
```diff
@@ -10,15 +10,17 @@
...
         "output_format": "{{ getv "/configuration/exporter/worker/input/output/output/format" "GPKG" }}",
-        "external_physical_path": "{{ getv "/configuration/exporter/worker/input/output/external/physical/path" ""}}",
+        "external_physical_path": "{{ getv "/configuration/exporter/worker/input/output/external/physical/path" "/ubuntu/outputs"}}",
         "internal_outputs_path": "{{ getv "/configuration/exporter/worker/input/output/internal/outputs/path" "./outputs"}}"
     },
     "logger": {
         "level": "{{ getv "/configuration/exporter/worker/logger/level" "INFO" }}",
-        "filename": "{{ getv "/configuration/exporter/worker/logger/filename" "ew_logs.log"}}"
+        "filename": "{{ getv "/configuration/exporter/worker/logger/filename" "ew_logs.log" }}",
+        "backup_count": {{ getv "/configuration/exporter/worker/backup/count" "10" }},
```
Comment (asafMasa) [inline confd/production.tmpl]: logger/backup/count
logger/max/bytes

File: confd/production.toml [hunk 2]
```diff
@@ -14,6 +14,8 @@ keys = [
     "/configuration/exporter/worker/input_output/internal_outputs_path",
     "/configuration/exporter/worker/logger/level",
     "/configuration/exporter/worker/logger/filename",
+    "/configuration/exporter/worker/backup/count",
```
Comment (asafMasa) [inline confd/production.toml]: logger/backup/count
logger/max/bytes

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: confd/production.toml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply confd/production.toml]: fixed in d72783d

File: confd/production.tmpl (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply confd/production.tmpl]: fixed in d72783d

**Review by asafMasa — APPROVED** [review]

**Pull request merged**